crates/neon/src/types_impl/mod.rs

```diff
@@ -55,14 +56,15 @@ pub use self::date::{DateError, DateErrorKind, JsDate};
 #[cfg_attr(docsrs, doc(cfg(all(feature = "napi-5", feature = "futures"))))]
 pub use self::promise::JsFuture;
 
-pub(crate) fn build<'a, T: Managed, F: FnOnce(&mut raw::Local) -> bool>(
+// TODO: Remove
```

Comment:
Hygiene again, can we turn the TODO into a followup issue? You can leave a comment still but link it to the issue.